### PR TITLE
Add wol package

### PR DESCRIPTION
**Pull request by gergesh**
wol implements Wake On LAN functionality in a small program. It wakes up hardware that is Magic Packet compliant.

### CODE REVIEW

**Comment by gergesh** [comment]
Little fuckup: I couldn't figure out how adding to the manpage works, so it's in the dependencies but doesn't actually do anything. The man file in the source is in /doc/wol.1

**Comment by uberhacker** [comment]
@gergesh: Thank you for the new package!  You could try `bash configure --help | less` and see what options are available.  There might be something like `--mandir` where you can set the man file location.

**Comment by gergesh** [comment]
I've fixed the manpage compression and removed a lot of unnecessary configure options, I think it should be okay now.

File: packages/wol.rb [hunk 1]
```diff
@@ -3,28 +3,20 @@
...
-            --prefix=#{CREW_PREFIX} \
-            --enable-gui=no \
-            --with-features=huge \
-            --without-x \
-            --disable-nls \
-            --enable-multibyte \
-            --with-tlib=ncurses \
-            --enable-perlinterp \
-            --disable-selinux"
+    system "./configure", "--prefix=#{CREW_PREFIX}", "--mandir #{CREW_PREFIX}/share/man"
```
Comment (uberhacker) [inline packages/wol.rb]: Missing `=` in between `--mandir` and `#{CREW_PREFIX}/share/man`?

Comment (gergesh) [reply packages/wol.rb]: Whoops, guess that's what happens when you make last-minute cosmetic changes without checking. I've fixed this and will do one last test to make sure everything functions properly.

**Comment by gergesh** [comment]
Program itself is working and locales are placed in the correct directory, but the manpages don't get installed and I have no idea why,

**Comment by uberhacker** [comment]
@gergesh: I think you have it correct.  Did you install the `manpages` package?  That lets you access the man stuff.  I tested and it looks fine:
```
$ crew files wol
(i) wol: Wake up hardware that is Magic Packet compliant
http://ahh.sourceforge.net/wol/
version 0.7.1
/usr/local/bin/wol
/usr/local/bin/wol-bootptab
/usr/local/bin/wol-dhcpdconf
/usr/local/info/wol.info.gz
/usr/local/share/locale/de/LC_MESSAGES/wol.mo
/usr/local/share/locale/es/LC_MESSAGES/wol.mo
/usr/local/share/locale/fr/LC_MESSAGES/wol.mo
/usr/local/share/locale/it/LC_MESSAGES/wol.mo
/usr/local/share/locale/sv/LC_MESSAGES/wol.mo
/usr/local/share/man/man1/wol.1.gz
Total found: 10
```

**Comment by gergesh** [comment]
H‌uh, guess that does it. Thanks a lot for the help!

**Pull request merged**